src/test/java/pageObjectTests/BaseTest.java

```diff
@@ -36,6 +36,6 @@ public void tearDown(ITestResult iTestResult) {
         if(iTestResult.getStatus()==iTestResult.FAILURE){
             ScreenShot.take(driver,iTestResult.getName());
         }
-        driver.close();
```

Comment:
why you deleted it?

Reply:
because we want to handle multiple windows